### PR TITLE
implement portal.get_default_language and portal.get_current_language

### PR DESCRIPTION
**Pull request by ebrehault**
according #291

### CODE REVIEW

**Comment by ebrehault** [comment]
jenkins: http://jenkins.plone.org/job/pull-request-5.0/785/

**Comment by ebrehault** [comment]
@jensens @hvelarde @gforcada @thet if you have a moment to review you

File: docs/portal.rst [hunk 1]
```diff
@@ -125,6 +125,42 @@ To display the date/time in a user-friendly way, localized to the user's prefere
...
+    # assert that the result is 'en'
+    self.assertEqual(lang, 'en')
+
+
+.. _portal_get_current_language_example:
+
+Get current language
+--------------------
+
+To get the currently negociated language, use :meth:`api.portal.get_current_language`.
```
Comment (adamcheasley) [inline docs/portal.rst]: Spelling mistake here. 'negociated', should be 'negotiated'.

File: src/plone/api/portal.py [hunk 2]
```diff
@@ -360,3 +360,36 @@ def set_registry_record(name=None, value=None, interface=None):
...
+
+def get_default_language():
+    """Return the default language.
+
+    :returns: language identifier
+    :rtype: string
+    :Example: :ref:`portal_get_default_language_example`
+    """
+    try:
+        from Products.CMFPlone.interfaces import ILanguageSchema
```
Comment (adamcheasley) [inline src/plone/api/portal.py]: When would importing `ILanguageSchema` raise an import error? Is that a difference between plone versions?

Comment (ebrehault) [inline src/plone/api/portal.py]: yes, in Plone 4 we use site_properties, not the registry

Comment (adamcheasley) [inline src/plone/api/portal.py]: Understood. I would consider moving the three lines below the import into an else clause for readability.

Comment (ebrehault) [inline src/plone/api/portal.py]: done, thanks

**Comment by adamcheasley** [comment]
:+1:

**Comment by gforcada** [comment]
Great!

**Pull request merged**